src/main/java/br/com/caelum/vraptor/panettone/VRaptorCompilationListener.java

```diff
@@ -19,7 +14,8 @@
 	public Variable[] getExtraInjections() {
 		return new Variable[] {
 				new Variable("javax.servlet.http.HttpServletResponse", "res"),
-				new Variable("br.com.caelum.vraptor.Result", "result")};
+				new Variable("br.com.caelum.vraptor.Result", "result"),
+				new Variable("br.com.caelum.vraptor.panettone.api.PrintEscaper", "printEscaper")};
```

Comment:
@guilhermesilveira I have a doubt. The declaration of this variable is in the correct place ? Or should be in PanettoneWalker?